packages/dx-react-chart/src/plugins/legend.jsx

```diff
@@ -1,17 +1,18 @@
 import * as React from 'react';
 import * as PropTypes from 'prop-types';
-
 import {
   Plugin,
   TemplateConnector,
   Template,
   TemplatePlaceholder,
 } from '@devexpress/dx-react-core';
+import { Marker } from '../templates/legend/marker';
+import { withComponents } from '../utils';
 
-export class Legend extends React.PureComponent {
+export class RawLegend extends React.PureComponent {
```

Comment:
What for do we export `RawLegend`? The same about `RawGrid`.

Reply:
Fixed (`RawGrid` had no `export`).